sdcm/kcl_thread.py

```diff
@@ -0,0 +1,136 @@
+import os
+import time
+import random
+import logging
+import uuid
+import threading
+
+from sdcm.stress_thread import format_stress_cmd_error, DockerBasedStressThread
+from sdcm.utils.docker_remote import RemoteDocker
+from sdcm.sct_events import StressEvent, InfoEvent, Severity
+from sdcm.cluster import BaseNode
+
+LOGGER = logging.getLogger(__name__)
+
+
+class KclStressEvent(StressEvent):
+    pass
+
+
+class KclStressThread(DockerBasedStressThread):  # pylint: disable=too-many-instance-attributes
+
+    def run(self):
+        _self = super().run()
+        # wait for the KCL thread to create the tables, so the YCSB thread beat this one, and start failing
+        time.sleep(120)
+        return _self
+
+    def build_stress_cmd(self):
+        if self.params.get('alternator_use_dns_routing'):
+            target_address = 'alternator'
```

Comment:
why not using 'alternator_use_dns_routing' instead of 'alternator' ?

Reply:
This name need to match the DNS server implementation we have, and it's hardcoded 'alternator' in that code